src/visual.ts

```diff
@@ -1262,11 +1266,16 @@ export class StreamGraph implements IVisual {
             dataLabelUtils.cleanDataLabels(this.svg);
         }
 
-        selectionMerged.attr("focusable", true);
-
         return selectionMerged;
     }
 
+    private updateLabelColorValueWhenMatchingWithBackgroundColor(colorPalette: ISandboxExtendedColorPalette, formattingSettings: StreamGraphSettingsModel) {
+        if ((colorPalette.background.value === "#FFFFFF" || colorPalette.background.value === "#000000")
+            && colorPalette.background.value === formattingSettings.enableDataLabelsCardSettings.color.value.value.toUpperCase()) {
```

Comment:
.attr("fill", this.colorHelper.isHighContrast ? this.colorHelper.getHighContrastColor("foreground", color) : color)

Reply:
fixed in 0e163a0